src/view/treeNodes/pullRequestNode.ts

```diff
@@ -89,6 +89,8 @@ export class PRNode extends TreeNode implements vscode.CommentingRangeProvider {
 				);
 			}
 
+			descriptionNode.updateContextValue();
```

Comment:
I see this is also called in the constructor of the `DescriptionNode`. Is the call here also needed? Is it enough that you also listen for `pullRequestModel.onDidChangeChangesSinceReview` in the `DescriptionNode`?